backend/config/curation-api.yml

```diff
@@ -427,7 +427,7 @@ paths:
         "413":
           $ref: "#/components/responses/413"
 
-  /v1/collections/{collection_id}/datasets/s3-upload-credentials:
+  /v1/collections/{collection_id}/s3-upload-credentials:
```

Comment:
hm... I don't think I agree. You're getting upload credentials that are specific  to a _Collection_, and not to a _Dataset_.

Reply:
Although I guess we aren't specifying a dataset in the path. Hm... ok I suppose this is alright.